src/store/wizardSlice.ts

```diff
@@ -40,6 +39,13 @@ type wizardState = {
     shareMethod: AwsShareMethod;
     source: V1ListSourceResponseItem | undefined;
   };
+  azure: {
+    shareMethod: AzureShareMethod;
+    tenantId: string;
+    subscriptionId: string;
+    source: string | undefined;
```

Comment:
If we are using `''` as the empty state for the source (which I think we should, it is consistent with Patternfly examples for typeahead selects) then the type here should only be `string`.